CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/UnlockVertexPositionsAction.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2010-2021 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.plugins.arrangements;
+
+import au.gov.asd.tac.constellation.graph.interaction.InteractiveGraphPluginRegistry;
+import au.gov.asd.tac.constellation.graph.node.GraphNode;
+import au.gov.asd.tac.constellation.plugins.PluginExecutor;
+import java.awt.event.ActionEvent;
+import javax.swing.AbstractAction;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionReferences;
+import org.openide.awt.ActionRegistration;
+import org.openide.util.NbBundle;
+
+/**
+ * Wrapper around UnlockVertexPositionsPlugin which is responsible for setting
+ * selected vertexes position_locked attributes to false to ensure the vertexes
+ * are moved when an arrangement plugin is run.
+ *
+ * @author serpens24
+ */
+@ActionID(category = "Arrange", id = "au.gov.asd.tac.constellation.plugins.arrangements.grid.UnlockVertexPositionsAction")
+@ActionRegistration(displayName = "#CTL_UnlockVertexPositionsAction",
+        iconBase = "au/gov/asd/tac/constellation/plugins/arrangements/resources/unlock.png",
+        surviveFocusChange = true)
+@ActionReferences({
+    @ActionReference(path = "Menu/Arrange", position = 9999),
+    @ActionReference(path = "Toolbars/Arrange", position = 9999),
+    @ActionReference(path = "Shortcuts", name = "C-A-U")
```

Comment:
Don't know how easy this would be but is there an option for possibly aligning this shortcut with the `LockPositionsAction`? L for lock and U for unlock makes sense but I think having C-L and C-A-U as mirror actions doesn't seem that intuitive (acknowledging C-A-L and C-U are already taken)

Reply:
C-U is for opening in a new sub graph which I'd like to keep. What is C-A-L used by? The other option could be that the C-L is used for both locking and unlocking using a toggling approach instead? This is what i did with the "freeze graph view" option.

Reply:
C-A-L is used to create a new layer for Layers view. While toggling could be an option, in this case I think it might be good to keep them separate. This is because when you make a selection you are probably wanting to either lock the selection or unlock it. If we go with a toggle, this won't be possible if you have a selection containing both locked and unlocked nodes (and having to check that all your nodes are one or the other may make for more manual work than desired)

Reply:
Hi @arcturus2  - Moving to pinned/unpinned hasnt made finding shortcuts any easier. I've delivered everything, but we need to find a useful set of shurtcuts. I've left them unchanged for now.